k8s_spark/parquet.spark.helm/templates/deployment.yaml

```diff
@@ -57,8 +57,10 @@ spec:
             {{- end }}
             - name: parquet_file_name
               value: {{ .Values.flask_env.parquet_file_name }}
+            {{- if ( index .Values "bitnami-spark" "enabled" ) }}
             - name: master_spark_url
-              value: {{ .Values.flask_env.master_spark_url }}
+              value: spark://{{ .Release.Name }}-bitnami-spark-master-0.{{ .Release.Name }}-bitnami-spark-headless:7077
```

Comment:
This is the existing value: 
```
spark://custom-spark-master-0.custom-spark-headless.bitnami-spark.svc.cluster.local:7077
```
Do we need to add the missing `bitnami-spark.svc.cluster.local`? Most likely we won't' need it, but I haven't actually tested it.

Reply:
Yes. I tested it locally, and it works. Thanks.